### PR TITLE
Add create_room and add_person

### PR DESCRIPTION
**Pull request by JayKay24**

### CODE REVIEW

File: controller.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+"""
+This module imports all the necessary classes to implement the controller aspect
+of the MVC framework.
+"""
+from dojo import Dojo
+from fellow import Fellow
```
Comment (Lemmah) [inline controller.py]: Good imports, I see why mine could have misbehaved a while back.

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+tornado==4.3
+traitlets==4.2.1
+twilio==5.7.0
+unicodecsv==0.14.1
+virtualenv==15.0.3
+Werkzeug==0.11.10
+xlrd==1.0.0
+XlsxWriter==0.9.2
+xlwings==0.7.2
+xlwt==1.1.2
```
Comment (Lemmah) [inline requirements.txt:163]: You have a lot of requirements. I suspect you are not using a **virtualenv** to work on your project. Please look into that, do not show **system-wide requirements**. Just show the end-user what they need to install to run your program.

Comment (JayKay24) [reply requirements.txt:163]: Yeah will definitely do that. I realized this while creating one with pip freeze > requirements.txt

**Comment by JayKay24** [comment]
Add functionality to randomize office and room allocations

**Pull request merged**